### DISCUSSION TITLE
PYTHON_PIP_VERSION

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#212] [Q&A]
Hi,
Is there any drawback (or maybe added value) from changing PYTHON_PIP_VERSION from 21.2.4 to latest (21.3.1)?
Same for PYTHON_SETUPTOOLS_VERSION to 58.3.0 from 57.5.0...?
I don't think other Python parameters (e.g. PYTHON_VERSION, GET PIP_URL, GET PIP_SH256) need to be touched...

am just addicted to be up to date :)

BR,

**Comment by mrlt8** [comment]
Changing those values won't update the container - you'd need to update the dockerfile and rebuild the container to do that.

**Comment by AdiAbuAli** [comment]
Right...i need to update those in dockerfile before re-build, but my question, is this a good practice?

**Reply by mrlt8** [reply]
pip is only used during build, so I don't see how it could help unless there's some major bug.
